### PR TITLE
Initial test review of ethblock.art

### LINKED ISSUE #21: Testing

### PR DESCRIPTION
**Pull request by mariuspod**
still in draft mode but a start for our review...

### CODE REVIEW

File: reviews/ethblock.art/Testing.md [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
-## Done by:
+## Commit-SHA: [eeda149192144921c4ebbd74efa6d1f2c4e85cd8](https://github.com/adrianleb/blockart-contracts/commit/eeda149192144921c4ebbd74efa6d1f2c4e85cd8)
+## Done by: [mariuspod](https://github.com/mariuspod)
 ### Reviewed by:
 
-
-### Initial observations
-* Tests only check for reverts
+## Summary
+* There are only 4 tests in total and 2 of them are not passing.
```
Comment (Marzel7) [inline reviews/ethblock.art/Testing.md:7]: We could suggest/write some tests to expand the test coverage

**Review by Marzel7 — APPROVED** [review]
Solid suggestions. 

To review the contracts effectively additional tests are required. We can expand the test coverage ourselves depending on the scope of the review. Let's discuss.

File: reviews/ethblock.art/Testing.md (hunk 1 above)
Comment (mariuspod) [reply reviews/ethblock.art/Testing.md:7]: actually I'm doing this under `Recommendations`, would you put it into the `Summary` as well ?

Comment (Marzel7) [reply reviews/ethblock.art/Testing.md:7]: Sure, will do.

**Review by Marzel7 — APPROVED** [review]
Excellent recommendations and well documented review.

**Pull request merged**